src/05-array-analysis.test.js

```diff
@@ -0,0 +1,10 @@
+import analyze from './05-array-analysis';
+
```

Comment:
Test failiure is likely because you put `import` here, instead of `require`

Reply:
Solved
![Screenshot from 2020-05-15 16-02-29](https://user-images.githubusercontent.com/5816027/82086870-86a67100-96c5-11ea-95cd-968265186514.png)